### ISSUE TITLE
Commands launched via sudo don't go through "tabelization/colorization"

### ISSUE THREAD

**Issue by alesito85** [nushell/nushell#8099]
### Describe the bug

Not sure how to express myself, but the screenshots below speak for themselves. I couldn't find an existing issue that would deal with that.

I've noticed that when commands are run via sudo, they don't go through "tabelization/colorization". For example regular ls output
<img width="457" alt="image" src="https://user-images.githubusercontent.com/4399118/219402477-7785b0db-1c8a-4a67-bc3a-1f01ced3b476.png">
Output of sudo ls
<img width="416" alt="image" src="https://user-images.githubusercontent.com/4399118/219402627-b7068cce-de71-448a-8175-fb400e06cd27.png">
Though I'm assuming this is for separate issue?

### How to reproduce

1. sudo ls
2. Observe output is standard `ls` output and not `nushell` type output

### Expected behavior

Output should go through the same process as usual, without sudo.

### Configuration

| key                | value                                                             |
| ------------------ | ----------------------------------------------------------------- |
| version            | 0.75.0                                                            |
| branch             |                                                                   |
| commit_hash        |                                                                   |
| build_os           | macos-aarch64                                                     |
| build_target       | aarch64-apple-darwin                                              |
| rust_version       | rustc 1.67.0 (fc594f156 2023-01-24) (built from a source tarball) |
| cargo_version      | cargo 1.67.0                                                      |
| pkg_version        | 0.75.0                                                            |
| build_time         | 2023-02-01 00:01:33 +00:00                                        |
| build_rust_channel | release                                                           |
| features           | database, default, trash, which, zip                              |
| installed_plugins  |                                                                   |

**Comment by fdncred** [comment]
I bet that's calling the operating systems `ls`. We do not have an `ls` that lists like that in nushell.

**Comment by alesito85** [comment]
That's likely, yeah. And it makes me think that sudo may have to be implemented in the same way, by wrapping it as an internal command to be handled along with the prompt.

**Comment by melMass** [comment]
I'm running into the opposite problem. `sudo -s` runs all command in nushell

**Comment by fdncred** [comment]
I'm not having any issues with `sudo ls` although I've noticed that it's not respecting my theme which makes me think it's launching another nushell on Windows, but that's just speculation.
![image](https://github.com/nushell/nushell/assets/343840/a777e945-32e8-4b93-bf33-32a7e6e8e5ca)

Of course, the sudo in Windows isn't officially released yet and also, i'm running gsudo/sudo for Windows. Other sudo's may vary.

On Ubuntu, with `sudo -s ls` I get the same looking table but my `$env.SHELL` is set to `nu`. Should we close this?

**Comment by melMass** [comment]
I think the op's issue is having the `SHELL` set to bash or zsh. So I would say yes, but let's see if it's that

**Comment by fdncred** [comment]
yup, if you don't have nushell as your login shell, the `sudo -s ls` won't work. It'll always run `ls` with your login shell.

I really have no idea how to integrate sudo with nushell more tightly, or even if it's possible.

**Comment by melMass** [comment]
From the man page using `$env.SHELL` is the way to integrate with `sudo -s`, not providing `-s` doesn't use a shell and for instance on macOS would run `/bin/ls`

**Comment by stevenxxiu** [comment]
On *Arch Linux*, `sudo --shell ls` isn't loading my *Nushell* config either.

It's not showing hidden files, showing different colors, etc.

**Comment by melMass** [comment]
Thats another issue all together I think, this table should help: https://github.com/nushell/nushell/issues/11847#issuecomment-1946049482

**Comment by stevenxxiu** [comment]
> Thats another issue all together I think, this table should help: [#11847 (comment)](https://github.com/nushell/nushell/issues/11847#issuecomment-1946049482)

Do you know where to put these flags so `sudo --shell ls` loads my *Nushell* config?

**Comment by melMass** [comment]
You unfortunately can't, you would have to run it without -s and call nu. 
I don't use default locations either and it's a bit of a pain

**Comment by stevenxxiu** [comment]
I just made this alias. It doesn't handle the many flags of `sudo`, but serves my use case:

```nu
def --wrapped sudo [...rest] {
  ^sudo env XDG_RUNTIME_DIR=/run/user/0 nu --login --commands ...$rest
}
```

**Comment by ask4fusora** [comment]
Can we upvote this issue.
